privacyidea/api/machineresolver.py

```diff
@@ -49,6 +49,7 @@
 
 @machineresolver_blueprint.route('/', methods=['GET'])
 @log_with(log)
+@prepolicy(check_base_action, request, ACTION.MACHINERESOLVERREAD)
```

Comment:
We also have a function ``get_resolver`` in this file, I think we should also add the decorator there?

Reply:
oh yes. Will do

Reply:
done in 003f1a44
(might be a good idea to resolve conversations)